### PR TITLE
IAEC-714 & 715

### PR DESCRIPTION
**Pull request by bleonard33**
Add the services classification to the HS and SITC hierarchies

### CODE REVIEW

File: product/HS/IntlAtlas/clean.py [hunk 1]
```diff
@@ -36,10 +38,22 @@
...
+    # Spread out services similarly to each set of exports but buffered further
+    service_starts = {
+        "section": 10,
+        "2digit": 400,
+        "4digit": 4000,
+    }
+    services = spread_out_entries(services, service_starts, h)
+
+    # Append to main table and sort on combined spread out indices
+    parent_id_table = parent_id_table.append(services).sort_index()
```
Comment (makmanalp) [inline product/HS/IntlAtlas/clean.py:55]: Sweet, didn't know about `sort_index`.

File: product/HS/IntlAtlas/in/Services_Hierarchy.tsv [hunk 2]
```diff
@@ -0,0 +1 @@
...
Other	Other	2digit	Other	Other	Other	Other	0
Travel services	Travel services	2digit	Travel services	Travel services	Travel	Travel	0
Transport services	Transport services	2digit	Transport services	Transport services	Transport	Transport	0
Communications	Communications	2digit	Communications	Communications	Communications	Communications	0
Insurance and financial services	Insurance and financial services	2digit	Insurance and financial services	Insurance and financial services	Insurance and finance	Insurance and finance	0
Other	Other	4digit	Other	Other	Other	Other	1
Travel services	Travel services	4digit	Travel services	Travel services	Travel	Travel	2
Transport services	Transport services	4digit	Transport services	Transport services	Transport	Transport	3
Communications	Communications	4digit	Communications	Communications	Communications	Communications	4
Insurance and financial services	Insurance and financial services	4digit	Insurance and financial services	Insurance and financial services	Insurance and finance	Insurance and finance	5
```
Comment (makmanalp) [inline product/HS/IntlAtlas/in/Services_Hierarchy.tsv]: Not a big deal but looks like the services input files have weird line terminators (`\r` / `^M`), would be nice to switch to using `\n` like everything else.

**Review by makmanalp — CHANGES_REQUESTED** [review]
Looks good, one minor change!

**Comment by bleonard33** [comment]
Verison bumped and EOL characters switched to unix instead of macos 👍

**Review by makmanalp — APPROVED** [review]

**Pull request merged**